frontend/packages/schema-model/src/lib/SchemaModel.ts

```diff
@@ -111,7 +111,13 @@ export class SchemaModel {
   }
 
   public getUniquePointer(pointer: string, uniqueParentPointer?: string): string {
-    if (!uniqueParentPointer || !isDefinitionPointer(pointer)) return pointer;
+    if (
+      !uniqueParentPointer ||
+      !isDefinitionPointer(pointer) ||
+      isDefinitionPointer(uniqueParentPointer) // Return original pointer if it's unique parent is a definition type (node in a Type)
```

Comment:
Jeg forstår ikke helt hvorfor dette er nødvendig. Løser dette feilen?

Skriv gjerne en test som gjenskaper feilen med koden slik den var før denne endringen.

Reply:
Det løser feilen ja. Klarer ikke helt å beskrive/forstå problematikken her. 
Skal undersøke litt videre og se om jeg får fikset det på en mer intuitiv måte. 
Gi beskjed hvis du får noen umiddelbare tanker basert på beskrivelsen under.

Kort av hva jeg logger når den fiksen ikke er med:
- navn1: element navn1 på root-nivå - har ikke parentPointer. 
- name1: element name1 i typen - der er typen parentPointer.

Klikker på element name1 to ganger så krasjer det. I siste del av viser den nå parentPointer med feil pointer (name1 er lagt til) og uniquePointer blir dermed også feil. 
![image](https://github.com/user-attachments/assets/4e15eabe-2327-4d19-9efc-7c28aa921c0e)

Reply:
For å gi en bedre forståelse, dette er logget etter man har fått uniquePointer fra `getUniquePointer` i filen SchemaNode.
![image](https://github.com/user-attachments/assets/c6a4e4fc-682b-4d64-828e-63e39e22e4b3)

Reply:
Jeg tar en titt på min egen maskin.